test/unit/html_attachment_test.rb

```diff
@@ -125,10 +132,12 @@ class HtmlAttachmentTest < ActiveSupport::TestCase
   end
 
   test "#url works with consultation public feedback for non-english locale" do
-    consultation = create(:consultation_with_public_feedback_html_attachment)
-    attachment = consultation.public_feedback.attachments.first
-    attachment.update!(locale: "fi")
-    assert_equal "/government/consultations/#{consultation.slug}/public-feedback/#{attachment.content_id}", attachment.url
+    Timecop.travel(11.years) do
+      consultation = create(:consultation_with_public_feedback_html_attachment)
+      attachment = consultation.public_feedback.attachments.first
+      attachment.update!(locale: "fi")
+      assert_equal "/government/consultations/#{consultation.slug}/public-feedback/#{attachment.content_id}", attachment.url
+    end
```

Comment:
Also worth adding a test that editions published before 29th November get the old behaviour.